shaders/src/light_directional.fs

```diff
@@ -58,13 +64,12 @@ void evaluateDirectionalLight(const MaterialInputs material,
         #if defined(MATERIAL_HAS_AMBIENT_OCCLUSION)
         visibility *= computeMicroShadowing(light.NoL, material.ambientOcclusion);
         #endif
-    } else {
+    }
 #if defined(MATERIAL_CAN_SKIP_LIGHTING)
+    if (visibility <= 0.0) {
```

Comment:
Visibility cannot be <= 0.0 if we don't enter the shadowing code, move inside the `if (light.NoL > 0.0) {`